Typescript/TS/tscript.ts

```diff
@@ -99,8 +99,8 @@ function func_2nd(){ //Button 2nd
     })
 }
 
-function eventList(e : any){ //Event listener callback function
-    let btnText = e.target.dataset.sign; // Using data attribute
+function eventList(e: any){ //Event listener callback function
+    let btnText: string = e.target.dataset.sign; // Using data attribute
```

Comment:
Instead of `string`, create and use a custom type alias for all possible values of `btnText`